config/environments/development.rb

```diff
@@ -21,15 +21,24 @@
 
   config.action_mailer.delivery_method = :smtp
 
-  config.action_mailer.smtp_settings = {
-    address: "localhost",
-    port: 1025
-  }
+  # config.action_mailer.smtp_settings = {
+  #   address: "localhost",
+  #   port: 1025
+  # }
```

Comment:
if you are not going to use config values, delete them, don't comment them.

that being said don't you want to use localhost for local email?